xxh3.h

```diff
@@ -1559,7 +1585,7 @@ XXH3_hashLong_64b_withSecret(const xxh_u8* XXH_RESTRICT input, size_t len,
 XXH_NO_INLINE XXH64_hash_t
 XXH3_hashLong_64b_withSeed(const xxh_u8* input, size_t len, XXH64_hash_t seed)
 {
-    XXH_ALIGN(8) xxh_u8 secret[XXH_SECRET_DEFAULT_SIZE];
```

Comment:
Note : I remember having measured some unexpected performance loss when increasing alignment requirement of this array. I could not explain it. But that's the reason why alignment remained at `8`.
_Maybe_ situation has changed, and this problem is no longer present, 
just present a set of benchmark to illustrate that outcome.

Reply:
on x86/x64, alignment greater than native stack alignment (16-byte for x64) will introduce an `and` instruction in the prolog of a function.

Reply:
That shouldn't make any measurable difference, aside from maybe requiring a frame pointer, which x86 is optimized to set up quickly

Reply:
on x64, alignment greater than 16 will also introduce `push rsp` and `pop rsp` instructions, these mem read/write operations seems measurable on short keys.